ios/FluentUI/PreviewCard/PreviewCard.swift

```diff
@@ -31,13 +33,16 @@ import SwiftUI
 /// View that represents the PreviewCard.
 public struct PreviewCard: View {
     var isElevated: Bool
+    var placeholderImage: UIImage
     @Environment(\.theme) var theme: FluentUIStyle
     @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
     /// Creates inner PreviewCard view.
     @ViewBuilder
     var innerContents: some View {
         HStack {
             Spacer()
+                /// Creates placeholder image over PreviewCard.
+            Image(uiImage: placeholderImage)
```

Comment:
Do we want to add some sort of clipping shape to this so that it'll also have rounded corners / be the same shape as the card?